client/src/test/java/com/linecorp/decaton/client/DecatonClientTest.java

```diff
@@ -39,8 +39,7 @@
 
 @ExtendWith(MockitoExtension.class)
 public class DecatonClientTest {
-    @Spy
-    private final DecatonClient<HelloTask> decaton = new DecatonClient<HelloTask>() {
+    private static class NoopClient implements DecatonClient<HelloTask> {
```

Comment:
Spying anonymous class fails on JDK8 with reflection error
https://github.com/line/decaton/actions/runs/8174863099/job/22350646261
<img width="1268" alt="image" src="https://github.com/line/decaton/assets/6070684/a6608124-d4ec-40b3-8fa0-4426ab086a3f">

I didn't want to dig into the detail as we likely drop JDK8-support in not-so-far-future so I just rewrote it with static class for now.